rollup.config.js

```diff
@@ -4,49 +4,57 @@ import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import pkg from './package.json';
 
-const config = {
-	input: 'src/index.js',
-	plugins: [
-		json(),
-		commonjs(),
-		buble({
-			target: {
-				node: 4
-			},
-			include: [
-				'src/**',
-				'node_modules/acorn-object-spread/**',
-				'node_modules/unicode-loose-match/**',
-				'node_modules/regexpu-core/**'
-			],
-			transforms: {
-				dangerousForOf: true
-			}
-		}),
-		resolve()
-	],
-	name: 'buble',
-	sourcemap: true
+const createConfig = ({
+	browser = false,
+	external = ['acorn', 'magic-string'],
```

Comment:
im using `acorn` here and not `acorn/dist/acorn.js` because it didnt have any meaning previously, acorn was inlined into the bundle (treated as internal) and resulted in duplication (buble has its own personal version of acorn bundled) - now it gets required nicely from cjs/es bundles